docs/admin_api/user_admin_api.rst

```diff
@@ -10,7 +10,8 @@ The api is::
 including an ``access_token`` of a server admin.
 The parameters ``from`` and ``limit`` are required only for pagination.
 By default, a ``limit`` of 100 is used.
-The parameter ``user_id`` can be used to filter by user id using an SQL regexp.
+The parameter ``user_id`` can be used to select only users with user ids that
+contain this value.
```

Comment:
I don't think this is what the implementation does, though.

If I make a request with `user_id=man`, it won't return `manuel`, even though `manuel` contains `man`.